Source/Atomic/Graphics/Direct3D11/D3D11Graphics.cpp

```diff
@@ -2807,6 +2807,12 @@ bool Graphics::GetMaximized()
 
     return SDL_GetWindowFlags(impl_->window_) & SDL_WINDOW_MAXIMIZED;
 }
+IntVector2 Graphics::GetMonitorResolution(int monitorId) const
+{
+    SDL_DisplayMode mode;
+    SDL_GetDesktopDisplayMode(monitorId, &mode);
+    return IntVector2(mode.w, mode.h);
+}
 // ATOMIC END
```

Comment:
Nice add, we really need a common Graphics base class to put this stuff in, there's a ticket for that... this is what we have to do for now